ui/js/directives/treeherder/clonejobs.js

```diff
@@ -249,18 +262,17 @@ treeherder.directive('thCloneJobs', [
             jobStatus.key = getJobMapKey(job);
             jobStatus.value = job.job_type_symbol;
             jobStatus.title = getHoverText(job);
-
+            jobStatus.visibleClass = job.visible ? "filter-shown" : "";
             if (thResultStatus(job) === "runnable") {
                 jobStatus.buildername = job.ref_data_name;
-                jobBtn = $(runnableJobBtnInterpolator(jobStatus));
-
                 if (ThResultSetStore.isRunnableJobSelected($rootScope.repoName,
                                                            job.result_set_id,
                                                            jobStatus.buildername)) {
-                    jobBtn.addClass("runnable-job-btn-selected");
+                    jobStatus.runnableClass = "runnable-job-btn-selected";
                 }
+                jobBtn = runnableJobBtnInterpolator(jobStatus);
             } else {
-                jobBtn = $(jobBtnInterpolator(jobStatus));
+                jobBtn = jobBtnInterpolator(jobStatus);
             }
 
             // If the job is currently selected make sure to re-apply
```

Comment:
@wlach, I finished with all the other issues, just a bit of this one left. I can't seem to understand what to store in place of `el`. There is the job ID, that's anyway stored under `repositories[repoName].lastJobObjSelected.id`. So I can use this ID maybe and fetch the element dynamically when needed. How can I use this ID with jQuery? I don't think `$("#" + job.id)` will work as this doesn't seem to be an HTML id